examples/rbd/snapshotclass.yaml

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
add a comment on the API version if you are using sidecar and controller v3.x this need to be betav1

Reply:
@Madhu-1 according to the validation results, If the controller+crds are 4.x and sidecar is 3.x, then we can still use `v1` APIs

In that case, will a matrix like the one below make sense?
```
# Here is the matrix showing what version of snapshot API should be used:
# v1.12 < k8s < v1.17, use v1alpha1
# v1.1 < snapshot-controller < v2.x, use v1alpha1 
# v1.17 =< k8s < v1.20, use v1betav1
# v2.x =< snapshot-controller < v4.x, use v1beta1
# k8s >= v1.20, use v1
# snapshot-controller >= v4.x, use v1
```

Reply:
alphav1 is not supported let remove it.

Reply:
> sounds good for a separate section in the upgrade doc.

@Madhu-1 updated now, PTAL, thanks!